python/paddle/fluid/tests/unittests/test_cryption.py

```diff
@@ -0,0 +1,178 @@
+#   Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import unittest
+import paddle.fluid as fluid
+from paddle.fluid import core
+
+
+class TestCryption(unittest.TestCase):
+    def _align_bytes_to_16(self, byte_str):
+        result_len = ((len(byte_str) + 15) // 16) * 16
+        pad_len = result_len - len(byte_str)
+        if pad_len > 0:
+            byte_str = bytearray(byte_str)
+            while pad_len > 0:
+                byte_str.append(0)
+                pad_len = pad_len - 1
+            byte_str = bytes(byte_str)
+        return byte_str, result_len
+
+    def test_cryption_in_memory_divisible_16(self):
+        input_str = str.encode("0123456789abcdef0123456789abcdef")
+
+        # get encrypt len
+        input_str, crypt_len = self._align_bytes_to_16(input_str)
+
+        cryptor = core.Cryption.get_cryptor()
+        encrypt_str = cryptor.encrypt_in_memory(input_str, crypt_len)
+        decrypt_str = cryptor.decrypt_in_memory(encrypt_str, crypt_len)
+        self.assertEqual(input_str, decrypt_str)
```

Comment:
this is not enough.  I suppose you should remove the padding part after decryption and compare it with the original raw input, right?

Reply:
Right, fixed